src/useValidation/useValidation.ts

```diff
@@ -29,6 +31,10 @@ export function useValidation<T>(
   const valid = computed(() => error.value === '')
   const invalid = computed(() => !valid.value)
   const pending = ref(false)
+  const initialValue = unref(value)
+  const touched = ref(false)
+  const dirty = ref(false)
```

Comment:
These can be easily determined by the user at implementation. And the composition doesn't use them internally. So I don't think these offer enough value.